apps/sms/views/conversation/js/conversation.js

```diff
@@ -1703,6 +1717,22 @@ var ConversationView = {
       pElement.setAttribute('data-l10n-id', 'no-attachment-text');
     }
 
+    var lateArrivalInfos = this.computeLateArrivalInfos(message, messageStatus);
+    if (lateArrivalInfos) {
+      var lateArrivalNoticeDOM = this.tmpl.lateArrivalNotice.prepare({})
```

Comment:
Actually it supports having no `options` but it does not support undefined `data` argument.